phpstan.neon

```diff
@@ -1,22 +1,26 @@
 parameters:
+    level: 8
+    paths:
+        - src/
+    bootstrapFiles:
+        - tests/bootstrap.php
     checkGenericClassInNonGenericObjectType: false
     checkMissingIterableValueType: false
     ignoreErrors:
-        - '#.+Generated\\Shared\\Transfer.+#'
-        - '#Parameter .+ of method .+ has invalid typehint type Orm\\Zed\\Payone\\Persistence\\.+#'
-        - '#Return typehint of method .+ has invalid type Orm\\Zed\\Payone\\Persistence\\.+#'
-        - '#Call to static method create\(\) on an unknown class Orm\\Zed\\.+\\Persistence\\.+#'
-        - '#Instantiated class Orm\\Zed\\Payone\\Persistence\\.+ not found.+#'
-        - '#Parameter .+ of method .+ has invalid typehint type Orm\\Zed\\Sales\\Persistence\\.+#'
-        - '#Call to method .+ on an unknown class Orm\\Zed\\.+\\Persistence\\.+#'
-        - '#PHPDoc tag @var for variable .+ contains unknown class Orm\\Zed\\.+\\Persistence\\.+#'
+        - '#Expression on left side of \?\? is not nullable\.#'
+        - '#PHPDoc type .+ of property .+ is not covariant with PHPDoc type .+ of overridden property .+#'
+        - '#Property .+ does not accept .+#'
```

Comment:
These are still to broad IMO.

Reply:
I can work on this place some more if it should.

Reply:
Fixed.